app/assets/stylesheets/buttons/primary_button.sass

```diff
@@ -0,0 +1,14 @@
+.primary-button
+  display: flex
+  flex-direction: row
+  align-items: center
+  span
+    &[data-icon-stack]
+      margin-right: 0.35em
+      color: $primary-color !important
+      .icon-color
+        color: $secondary-font-color !important
+  a
+    color: $primary-color !important
+    text-decoration: underline !important
```

Comment:
This probably needs `cursor: pointer` too, by virtue of some of the `ng-href`s being empty.